pkg/operator/staticpod/controller/guard/bindata/bindata.go

```diff
@@ -0,0 +1,313 @@
+// Code generated for package bindata by go-bindata DO NOT EDIT. (@generated)
```

Comment:
Oh my, we got rid of go-bindata from all the operators but library-go is still left with one :disappointed: we'll should fix that...

Reply:
That will require more changes. Better to do it in another PR.

Reply:
Definitely a separate PR, just an observation :wink: